modules/culling/src/lib/axis-aligned-bounding-box.js

```diff
@@ -135,4 +85,36 @@ export default class AxisAlignedBoundingBox {
 
     return INTERSECTION.INTERSECTING;
   }
+
+  // Computes the estimated distance from the closest point on a bounding box to a point.
+  distanceTo(point) {
+    return Math.sqrt(this.distanceSquaredTo(point));
+  }
+
+  // Computes the estimated distance squared from the closest point on a bounding box to a point.
+  // A simplified version of OrientedBoundingBox.distanceSquaredTo
+  distanceSquaredTo(point) {
+    const offset = scratchVector.from(point).subtract(this.center);
+    const {halfDiagonal} = this;
+
+    let distanceSquared = 0.0;
+    let d;
+
+    d = Math.abs(offset.x) - halfDiagonal.x;
```

Comment:
Hmm, does it really work to do this component-wise? Can we skip calculating the intersection point with the bounding box? Did you peek at the CesiumJS implementation?

Don't have time to think it through. And admittedly, not terribly important - even if not precisely correct, this is likely good enough for almost all uses.

Reply:
Since it's axis aligned, e.g. `xAxis` is `[1, 0, 0]`, and `offset.dot(xAxis) === offset.x`